types/schemas/prototypeApplication/data/Applicant.ts

```diff
@@ -0,0 +1,167 @@
+import {Date, Email} from '../../../shared/utils';
+import {PrimaryApplicationType} from '../enums/ApplicationType';
+import {UserRoles} from './User';
+
+export type ApplicantBase = BaseApplicant | Agent;
+
+export type BaseApplicant = ContactDetails & {
+  /**
+   * @Description The type of applicant
+   */
+  type: 'individual' | 'company' | 'charity' | 'public' | 'parishCouncil';
+  /**
+   * @description Address information for the applicant
+   */
+  address: ApplicantAddress;
+  /**
+   * @description Contact information for the site visit
+   */
+  siteContact: SiteContact;
+};
+
+export interface Agent extends BaseApplicant {
+  /**
+   * @description Contact information for the agent or proxy
+   */
+  agent: ContactDetails & {address: UserAddress};
+}
+
+export type SiteContact = {role: UserRoles} | SiteContactOther;
+
+export interface SiteContactOther {
+  role: 'other';
+  name: string;
+  email: string;
+  phone: string;
+}
+
+export type ContactDetails = {
+  name: {
+    title?: string;
+    first: string;
+    last: string;
+  };
+  email: Email;
+  phone: {
+    primary: string;
+  };
+  company?: {
+    name: string;
+  };
+};
+
+export type UserAddress = {
+  line1: string;
+  line2?: string;
+  town: string;
+  county?: string;
+  postcode: string;
+  country?: string;
+};
+
+export type ApplicantAddress =
+  | {sameAsSiteAddress: true}
+  | ApplicantAddressNotSameSite;
+
+export interface ApplicantAddressNotSameSite extends UserAddress {
+  sameAsSiteAddress: false;
+}
+
+export type OwnershipInterest = 'owner' | 'tenant' | 'occupier' | 'other';
+
+export interface BaseOwners {
+  name: string;
+  address: UserAddress | string;
+  interest?: OwnershipInterest;
+}
+
+export interface OwnersNoticeGiven extends BaseOwners {
+  noticeGiven: true;
+}
+
+export interface OwnersNoNoticeGiven extends BaseOwners {
+  noticeGiven: false;
+  noNoticeReason: string;
+}
+
+export interface OwnersNoticeDate extends BaseOwners {
+  noticeDate: Date;
+}
```

Comment:
`Ownership` is a nice migration story:

Our original `application` types used code comments to note which shape of owners we expected for various application types, ultimately justifying a type that was too-generic and made _all_ of these options possible: https://github.com/theopensystemslab/digital-planning-data-schemas/blob/main/types/schemas/application/data/Applicant.ts#L88-L101

Now the `LDCApplicant` definition and so on specify _only_ the ownership variation it _actually_ expects!